generators/vsix-vs-win/BotBuilderVSIX-V4/UncompressedProjectTemplates/CoreBot/CoreBot.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Generated with `dotnet new corebot` vX.X.X
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.BotBuilderSamples
```

Comment:
This namespace need to be templatized acrosss all the .cs files. For example:
namespace $safeprojectname$

You can see an example here: https://github.com/Microsoft/BotBuilder-Samples/blob/master/generators/vsix-vs-win/BotBuilderVSIX-V4/UncompressedProjectTemplates/EchoBot/CounterState.cs

Reply:
Feedback applied